src/main/groovy/com/blackducksoftware/integration/hub/detect/interactive/mode/DefaultInteractiveMode.java

```diff
@@ -55,15 +55,24 @@ public void interact() {
                     setPropertyFromQuestion("hubApiToken", "What is the hub API token?");
                 } else {
                     setPropertyFromQuestion("hubUsername", "What is the hub username?");
-                    setPropertyFromSecretQuestion("hubPassword", "What is the hub password?");
+
+                    final Boolean setHubPassword = askYesOrNoWithMessage("Would you like to set the hub password?",
+                            "WARNING: If you choose to save the settings, this password will be stored in plain text. You can set this password as an environment variable BLACKDUCK_HUB_PASSWORD.");
```

Comment:
I think we should word the second sentence here a little differently. "You can set this password as the environment variable BLACKDUCK_HUB_PASSWORD"